integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -95,6 +95,7 @@ else
                 "k8s-shared-volume.bats" \
                 "k8s-volume.bats" \
                 "k8s-nginx-connectivity.bats" \
+				"k8s-ro-volume.bats" \
```

Comment:
nit: the line is not aligned. Can you replace the tabs with spaces?

Reply:
fixed